servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -367,3 +369,10 @@ def _parse_datetime(date_string):
 
 def _get_base_url():
     return current_app.config['cromwell_url']
+
+
+def _format_query_labels(orig_query_labels):
+    query_labels = {}
+    for key, val in orig_query_labels.items():
+        query_labels[key] = urllib.unquote(val).decode('utf8')
```

Comment:
Are you sure this `decode()` is necessary? 

It seems that `unquote()` returns a string, while `decode()` [operates on bytes](https://docs.python.org/3/library/stdtypes.html#bytes.decode).

I think we may be taking the UTF-8 encoded string returned by `unquote()`, casting it to an array of bytes, and then decoding it into the same UTF-8 string.

Reply:
removed `decode()`, and it still worked